### DISCUSSION TITLE
How to change the color/size of the href link of the title in each pages ?

### DISCUSSION THREAD

**Discussion by emmakopp1** [academicpages/academicpages.github.io#2448] [Q&A]
I want to change the color and size of the clickable links for the titles on each page (e.g., Publications, Talks, etc.). Since they all use the same font, I assume I can apply the changes to all items automatically?

**Comment by rjzupkoii** [comment]
Most of the basic settings for the fonts, colors, and so forth are in `_sass/_variables.scss` and can be changed there.

**Reply by debmallyachanda** [reply]
Is it possible to change color of a specific tab?

**Reply by rjzupkoii** [reply]
@debmallyachanda Depending on what you are trying to do, you can use [HTML Styles](https://www.w3schools.com/html/html_styles.asp) to change things on a case-by-case basis.

**Reply by debmallyachanda** [reply]
To be more specific, I want to change the font color of the tab name that appears on the top of the homepage, say font color of 'Publications'  tab
